main.go

```diff
@@ -95,6 +97,8 @@ func initFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&syncPeriod, "sync-period", 10*time.Minute, "The minimum interval at which watched resources are reconciled (e.g. 10m)")
 	fs.IntVar(&webhookPort, "webhook-port", 9443, "Webhook Server port, disabled by default. When enabled, the manager will only work as webhook server, no reconcilers are installed.")
 	fs.DurationVar(&reconcileTimeout, "reconcile-timeout", reconciler.DefaultLoopTimeout, "The maximum duration a reconcile loop can run (e.g. 90m)")
+	fs.Float32Var(&restConfigQPS, "kube-api-qps", 20, "Maximum queries per second from the controller client to the Kubernetes API server. Defaults to 20")
```

Comment:
Where is the default here and for the other parameter coming from? Does it match what you get when you don't specify it at all?

Asking because I'm wondering if we should default it to zero so that we fall back to whatever semantics are associated with the default value in client-go / the manager.

Reply:
No, it matches what CAPI uses as default. My intention was to keep it consistent with that.
The client-go default is 5 for qps and 10 for burst.